flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -213,6 +213,12 @@ static void processConfigOptions(String rootDir, String module, String packageNa
 		}
 	}
 
+	private static boolean isValidConfigOption(Field field) {
```

Comment:
I'm not fond of the naming as a deprecated options is still valid.

Maybe split this into `isConfigOption` that does the `ConfigOption` check and `shouldBeDocumented` that does the deprecation/exclusion checks.

Reply:
Will split it up.